balboa-service-jms/src/main/java/com/socrata/balboa/jms/BalboaJms.java

```diff
@@ -59,7 +60,17 @@ public static void main(String[] args) throws Exception {
 
         log.info("Receivers starting, awaiting messages.");
         DataStore ds = DefaultDataStoreFactory.get();
-        ActiveMQReceiver receiver = new ActiveMQReceiver(servers, channel, threads, ds);
+
+        DataStore hawkular = new HawkularMetricStore(
+                new URI("http://10.110.35.139:14996/"),
```

Comment:
eh... it would be great if these could be configurable.  But I know that the configuration system for balboa is all sorts of 💩, so, I would understand if that is difficult.